controls/emet.rb

```diff
@@ -0,0 +1,16 @@
+# encoding: utf-8
+
+title 'Ms EMET'
+
+control 'EMET-1' do
+  impact 0.7
+  title 'Ms EMET is running'
+  desc 'EMET process monitoring is active'
+  describe processes('EMET_Service.exe') do
```

Comment:
Is this working on all Windows versions? Otherwise we should use `only_if { }` to restrict the windows release